Added badge (#2)
corgi-vision/skin-cancer-detection: README.md

### Kaggle ISIC 2024 challange 

> In this project, we will develop image-based algorithms to identify histologically confirmed skin cancer cases with single-lesion crops from 3D total body photos (TBP). The image quality resembles close-up smartphone photos, which are regularly submitted for telehealth purposes. Your binary classification algorithm could be used in settings without access to specialized care and improve triage for early skin cancer detection. We had a large dataset of images, and for every image we are able to use some metadata about the patient, leisure and the picture. We choosed some, which ones we thought might be important, as it shown below

Reviewer: ```suggestion
In this project, we will develop image-based algorithms to identify histologically confirmed skin cancer cases with single-lesion crops from 3D total body photos (TBP). The image quality resembles close-up smartphone photos, which are regularly submitted for telehealth purposes. We have a large dataset of images, and for every image, we can use some metadata about the patient and the leisure.
```

In this project, we will develop image-based algorithms to identify histologically confirmed skin cancer cases with single-lesion crops from 3D total body photos (TBP). The image quality resembles close-up smartphone photos, which are regularly submitted for telehealth purposes. Your binary classification algorithm could be used in settings without access to specialized care and improve triage for early skin cancer detection. We had a large dataset of images, and for every image we are able to use some metadata about the patient, leisure and the picture. We choosed some, which ones we thought might be important, as it shown below

### Used metadata:

> | Name                        | Description                                                  | Regularization method |

Reviewer: Szerintem jobb lenne egy külön file-ban (pl. info.md) tárolni ezt a táblázatot, hogy jobban átlátható legyen a readme

Author: Jogos

| tbp_lv_radial_color_std_max | Color asymmetry, a measure of asymmetry of the spatial distribution of color within the lesion. This score is calculated by looking at the average standard deviation in L*A*B* color space within concentric rings originating from the lesion center. Values range 0-10.+ | Min-Max Scaling           |
| tbp_lv_symm_2axis           | Border asymmetry; a measure of asymmetry of the lesion's contour about an axis perpendicular to the lesion's most symmetric axis. Lesions with two axes of symmetry will therefore have low scores (more symmetric), while lesions with only one or zero axes of symmetry will have higher scores (less symmetric). This score is calculated by comparing opposite halves of the lesion contour over many degrees of rotation. The angle where the halves are most similar identifies the principal axis of symmetry, while the second axis of symmetry is perpendicular to the principal axis. Border asymmetry is reported as the asymmetry value about this second axis. Values range 0-10.+ | Min-Max Scaling           |
| tbp_lv_symm_2axis_angle     | Lesion border asymmetry angle.+                              | Min-Max Scaling           |

> Files in the repo: 

Reviewer: Itt egyelőre csak a notebook-ot említeném meg. pl
The notebook for exploring and preprocessing the data: explore.ipynb

Author: @benjaminChonto szóval, hogy miket választottunk egyenlőre nem?

Reviewer: Itt a `Files in repo` sorra gondolok, hogy azt soroljuk fel, amit át kell nézniük a beadás során, ami egyelőre csak az explore.ipynb notebook
